CONTRIBUTING.md

```diff
@@ -26,12 +26,30 @@ Here are a few things you can do that will increase the likelihood of your pull
 
 This section describes a typical sequence performed when developing locally. Full details of available tooling are available in the next section on [Automated And Manual Testing](#automated-and-manual-testing).
 
-1.  Export necessary configuration for authenticating your provider with GitHub
+### Local Development Setup
+
+Once you have the repository cloned, there's a couple of additional steps you'll need to take. Since most of the testing is acceptance or integration testing, we need to manipulate GitHub resources in order to run it. Useful setup steps are listed below:
+
+- If you haven't already, create a GitHub organization you can use for testing.
+  - Optional: some may find it beneficial to create a test user as well in order to avoid potential rate-limiting issues on your main account.
+  - See [here](#github-organization) for further instructions on creating an organization.
```

Comment:
```suggestion
- If you haven't already, [create a GitHub organization you can use for testing](#github-organization).
  - Optional: some may find it beneficial to create a test user as well in order to avoid potential rate-limiting issues on your main account.
```